.github/workflows/rust.yml

```diff
@@ -49,12 +53,12 @@ jobs:
       run: cargo build --target ${{ matrix.rust.target }}
 
     - name: Test with default CPU features + No Default Cargo Features
-      if: matrix.rust.target == 'i586-pc-windows-msvc' || matrix.rust.target == 'i686-pc-windows-msvc' || matrix.rust.target == 'x86_64-pc-windows-msvc' || matrix.rust.target == 'wasm32-wasi'
+      if: matrix.rust.target == 'i586-pc-windows-msvc' || matrix.rust.target == 'i686-pc-windows-msvc' || matrix.rust.target == 'x86_64-pc-windows-msvc' || matrix.rust.target == 'aarch64-apple-darwin' || matrix.rust.target == 'wasm32-wasi'
```

Comment:
These lines cover all of the systems in the matrix above, so I think they could probably go away entirely instead of just adding a new platform to the list here and in each below.

Reply:
Removed in #171.